public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/front-end.scss

```diff
@@ -0,0 +1,11 @@
+/*
+ * todo
+ *
+ * does this file get bundled into blocks.min.css? if so, that could mess up the order b/c it might be
+ * re-including these after edit.scss does.
+ * 		maybe need to exclude front-end.scss, or just only include edit.scss
```

Comment:
blocks.min.css is generated from the blocks entry point, and the only place this is imported is front-end.js, so no — it should not be included in the blocks.min.css.

Reply:
👍